### DISCUSSION TITLE
Using truncate marker, Tina does not allow saving

### DISCUSSION THREAD

**Discussion by BattleOfPlassey** [tinacms/tinacms#3919] [Help] [answered]
If you have an HTML comment in the body, Tina does not allow saving.

```
<!--truncate--> 
````
I raised the same issue in Tinasaurus in the link below. Coming from forestry i faced no such issue.
https://github.com/tinacms/tinasaurus/issues/4

Also, I believe there is an open issue for it.
https://github.com/tinacms/tinacms/issues/3213

**Comment by logan-anderson** [comment]
@BattleOfPlassey What version of TinaCMS are you using and are you using MDX or markdown?

**Reply by BattleOfPlassey** [reply]
"dependencies": {
    "@docusaurus/core": "^2.4.0",
    "@docusaurus/preset-classic": "^2.4.0",
    "@docusaurus/theme-search-algolia": "^2.4.0",
    "@mdx-js/react": "^1.6.22",
    "@tinacms/cli": "^1.2.2",
    "clsx": "^1.2.1",
    "prism-react-renderer": "^1.3.5",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "tinacms": "^1.2.2",
    "title": "^3.5.3"
  },

**Comment by BattleOfPlassey** [comment, marked as answer]
@logan-anderson Thanks buddy, i just realized we cannot have `< >` symbols in mdx files afterall they are used for JSX code .  I just did a workaround. I went back to using md files and changing the config types of all fields from mdx to md.
